app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+key = os.getenv("API_KEY")
```

Comment:
Code Style issue: The variable 'key' is essentially a constant because its value is not meant to change during the execution of the script. Therefore, it should be named in uppercase, like 'KEY', as per the Python style guide for constants.